CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
+# 0.6.0
+
+* Feature - Added support for [AWS SSO](https://aws.amazon.com/single-sign-on/) ([#229](https://github.com/awslabs/amazon-ecr-credential-helper/issues/229))
+* Feature - Added support for [AWS SDK for Go v2](https://aws.github.io/aws-sdk-go-v2/docs/). AWS SDK for Go V2 offers significant improvments in CPU and memory utilization in addition to features such as modularization, intuitive configuration, and more simplified API client methods ([#282](https://github.com/awslabs/amazon-ecr-credential-helper/issues/282))
```

Comment:
```suggestion
```

Most of these things don't really matter to an end user.  If we have measurements showing improvements in CPU and memory utilization for the credential helper that would be worth talking about, but I don't know if we've actually measured this.

Reply:
I was more using the features listed in the AWS SDK for Go v2 release documentation. The [original issue](https://github.com/awslabs/amazon-ecr-credential-helper/issues/282) referenced being able to assumed roles via EC2 instance metadata, but the change really seems more than just that single feature to me.

Reply:
According to https://github.com/awslabs/amazon-ecr-credential-helper/issues/282

> Most of all I would like to be able to assume roles via EC2 instance metadata (this works on v2 but not v1).

Can we mention that instead? Not really sure why that is tied to SDK change tbh though.

Reply:
Okay, I think I understand better now. I'll focus on that specific feature rather than documenting adding support for the entire SDK.

Reply:
The things listed in the changelog for the credential helper should be things that matter to someone using the credential helper.  If the v2 SDK added features that they can use, the specific features/added functionality is what we want to highlight.